includes/functions.php

```diff
@@ -1230,4 +1246,17 @@ function function_check($function) {
 
 function get_last_commit() {
     return `git log -n 1|head -n1`;
-}//end get_last_commit 
+}//end get_last_commit
+
+function snmpTransportToAddressFamily($transport) {
```

Comment:
@laf `snmpTransportToAddressFamily` is a new function introduced by this PR, so I understand @f0o's wish to have it documented.
Should get around to that within about a day.